conda_forge_tick/update_upstream_versions.py

```diff
@@ -189,7 +189,14 @@ def update_upstream_versions(gx, sources=(PyPI(), CRAN(), Github(),
                                           RawURL()
                                           )):
     for node, attrs in gx.node.items():
-        attrs['new_version'] = get_latest_version(attrs, sources)
+        try:
+            attrs['new_version'] = get_latest_version(attrs, sources)
+        except Exception as e:
+            logger.warn('Error getting uptream version of {}: '
+                        '{}'.format(node, e))
+            with open('upstream_bad', 'a') as f:
+                f.write('{}: Error getting upstream version\n'.format(node)
+            attrs['new_version'] = False
         logger.info('{} - {} - {}'.format(node,
```

Comment:
You might put this inside the try or as an else. That way we don't have multiple messages per feedstock (we already have a lot of messages)